doc/quickstart.md

```diff
@@ -0,0 +1,111 @@
+# vfkit quick start
+
+## Introduction
+
+vfkit is a macOS command line hypervisor.
+It uses [Apple's Virtualization Framework](https://developer.apple.com/documentation/virtualization?language=objc) to run virtual machines.
+You start a virtual machine by running vfkit with a set of arguments describing the virtual machine configuration/hardware.
+When vfkit stops, the virtual machine stops running.
+It runs on both x86_64 and aarch64 Macs.
+
+
+## Installation
+
+You can get vfkit either by downloading it from [its release page](https://github.com/crc-org/vfkit/releases), or get it from [brew](https://brew.sh/):
+```
+# Only the first time
+$ brew tap cfergeau/crc
+
+$ brew install vfkit
+```
+
+
+## Quick start
+
+### Getting a disk image
+
+Your virtual machine will need an operating system to run, so you need to download a disk image first.
+The image needs to be in the raw or iso format, qcow2 or VirtualBox images cannot be used by vfkit.
+
+For example, Fedora images can be downloaded with:
+```
+# For Apple silicon Macs
+$ curl -L -O https://download.fedoraproject.org/pub/fedora/linux/releases/38/Cloud/aarch64/images/Fedora-Cloud-Base-38-1.6.aarch64.raw.xz
+$ xz -d ./Fedora-Cloud-Base-38-1.6.aarch64.raw.xz
+$ cp -c Fedora-Cloud-Base-38-1.6.aarch64.raw Fedora-Cloud-Base-38.raw
+
+# For Intel Macs
+$ curl -L https://download.fedoraproject.org/pub/fedora/linux/releases/38/Cloud/x86_64/images/Fedora-Cloud-Base-38-1.6.x86_64.raw.xz
+$ xz -d ./Fedora-Cloud-Base-38-1.6.x86_64.raw.xz
+$ cp -c Fedora-Cloud-Base-38-1.6.x86_64.raw Fedora-Cloud-Base-38.raw
```

Comment:
why not `mv` ? the resulting copy is only to use a consistent name in the next paragraphs.

Reply:
It's a good point that `cp -c` might be "too advanced" for this guide. However, to me it's a very neat feature to be aware of, it's macOS-specific so users are likely not to be aware of it, and it allows to have the equivalent of qcow2 backing files. You keep a base image, and you have an overlay with your changes on top of it. Maybe move this to its own paragraph with more details? Or to the reference documentation ?

Reply:
I have left this unchanged in the update I just made, but will do more changes now.

Reply:
I've addressed this now, only using `mv` in the quick start document, and pointing at a more detailed doc about `cp -c` and `truncate` and corresponding syscalls.